pkg/resource/compare/defaults.go

```diff
@@ -176,7 +176,7 @@ func equalDeploymentConfigs(deployed client.Object, requested client.Object) boo
 	pairs = append(pairs, [2]interface{}{dc1.Spec, dc2.Spec})
 	equal := EqualPairs(pairs)
 	if !equal {
-		logger.Info("Resources are not equal", "deployed", deployed, "requested", requested)
+		logger.Debugf("Resources are not equal", "deployed", deployed, "requested", requested)
```

Comment:
For case when resources or objects are not equal. Can we keep there at least some info output. Otherwise it might be hard for users to spot what is wrong.
If the debug output is not allowed, it would be good to print some info log (or warning) as well. So it can be debug easier. I think it should be okay to just print message like "Resources are not equal. For more details use debug option for your Operator."

Reply:
@jakubschwan thank you for the suggestion, I agree with your concern and I'll do the change